roles-in-tech/developer-advocate.md

```diff
@@ -2,89 +2,146 @@
+It is important to report to your product teams and engineers. This includes feedback about your own product, but also what positive and negative feedback the competition got. This also includes introducing the teams to people and to communicate with in case they want to collaborate on a new feature.
+Amplify messaging of internal teams:
+Your colleagues bring great content, it is your job to give it reach. Amplify their messages by spreading them far and wide and explaining them in an understandable fashion to different audiences. Tell other influencers about the direct information channels your teams have.
+Coach and promote internal talent:
+It is great that you can represent your company, but sometimes the voice of someone working directly on a product has more impact. Coach people in the company and promote their presence on social media and at events. Connect conference organisers and internal people — make sure to check with their manager to have them available. Help people present and prepare materials for outreach.
+Report on events and success of campaigns:
+Being at an event or running a campaign should be half the job. The other half is proving to the people in the company that it was worth it. Make sure to collect questions you got, great things you learned and find a good way to communicate them to the teams they concern. Keep a log of events that worked and those who didn’t — this is great information for marketing when it comes to sponsorship.
+Help organising events:
+It is important to be involved in the events your company organises. Try not to get roped into presenting at those — instead offer to coach people to be there instead. Offer to be on the content board of these events — you don’t want to be surprised by some huge press release that says something you don’t agree with. Use your reach to find external speakers for your events.
+Work with PR, legal and marketing:
+Your work overlaps a lot with those departments. Be a nice colleague and help them out and you get access to more budget and channels you don’t even know about. Make sure that what you say or do in your job is not causing any legal issues.
+Give constructive feedback to the product teams and get questions answered:
+You’re seen as a approachable channel for questions about your products. Make sure that requests coming through you are followed up swiftly. Point out communication problems to your internal teams and help them fix those. Use external feedback to ask for improvements of your own products. It is easy to miss problems when you are too close to the subject matter.
+Collate outside feedback and convert to constructive feedback:
+Your colleagues are busy building products. They shouldn’t have to deal with angry feedback that has no action items in it. Your job is to filter out rants and complaints and to analyse the cause of them. Then you report the root issue and work with the teams how to fix them.
 
 ### Materials
```

Comment:
“Materials” is for assigned materials, like the slideshow, video, or what you are asking them to do in the outline. If it is optional, put it in a “supplemental resources “ section.